public/src/components/Subreddit/InnerHeaderComponent.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+export default class InnerHeaderComponent extends React.Component {
+    getHeaderStyle() {
+        const {
+            banner_background_image,
+            banner_background_color,
+            key_color,
+        } = this.props;
+        const backgroundImage = banner_background_image;
+        const color = banner_background_color || key_color;
+
+        return {
+            backgroundColor: color || 'none',
+            backgroundImage: backgroundImage ? `url(${backgroundImage})` : 'none',
+        };
+    }
+
+    getImage() {
+        const { icon_img } = this.props;
+        return icon_img
+            ? (
+                <img className="subreddit-header__image"
+                     src={icon_img}
+                     alt="Subreddit icon"
+                />
+            )
+            : null;
```

Comment:
Use arly return in these cases? Otherwise it quickly becomes an indentation nightmare

```js
const { icon_img } = this.props;

if (!icon_img) {
    return null;
}

return (
    <img
        className="subreddit-header__image"
        src={icon_img}
        alt="Subreddit icon"
    />
)
```